src/Libraries/Microsoft.Extensions.Http.Resilience/Resilience/ResilienceHandlerContext.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Extensions.Http.Resilience.Internal;
+using Polly.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Http.Resilience;
+
+/// <summary>
+/// The context used when building a resilience strategy HTTP handler.
+/// </summary>
+public class ResilienceHandlerContext
```

Comment:
Since there isn't a public constructor and so the type can't be extended, make it sealed.